recipes/backport-cpp/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class BackportCppRecipe(ConanFile):
+    name = "backport-cpp"
+    description = "An ongoing effort to bring modern C++ utilities to be compatible with C++11"
+    topics = ("conan", "backport-cpp", "header-only", "backport")
+    homepage = "https://github.com/bitwizeshift/BackportCpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    generators = "cmake"
+    no_copy_source=True
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "BackportCpp-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BACKPORT_COMPILE_UNIT_TESTS"] = False
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib"))
```

Comment:
For header only I would suggest just a copy (it's faster than configuring CMake)

⚠️ not tested

```suggestion
    def package(self):
        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
        self.copy(os.path.join("include", "**", "*.hpp"), dst="include", src=self._source_subfolder) # you may need to play with keep_path
```

Reply:
I see. For this library this is indeed possible, just copying is mentioned as one way to do it in projects readme. My nature is more to go the safe way, for the case it there a generated headers for example.

Reply:
Fair enough. case by case... I try to only add projects I am familiar with to avoid missing details like that